go/oasis-test-runner/scenario/e2e/debond.go

```diff
@@ -37,7 +38,41 @@ func (s *debondImpl) Fixture() (*oasis.NetworkFixture, error) {
 	f.Network.EpochtimeMock = true
 
 	// Enable some features in the staking system that we'll test.
-	f.Network.StakingGenesis = "tests/fixture-data/debond/staking-genesis.json"
+	f.Network.StakingGenesis = &staking.Genesis{
+		Parameters: staking.ConsensusParameters{
+			CommissionScheduleRules: staking.CommissionScheduleRules{
+				RateChangeInterval: 10,
+				RateBoundLead:      30,
+				MaxRateSteps:       4,
+				MaxBoundSteps:      12,
+			},
+		},
+		TotalSupply: stakingTests.QtyFromInt(1000),
```

Comment:
There's also that `quantity.NewFromUint64` that we could use instead of importing tests (and `QtyFromInt` could just be removed).